CHANGELOG.md

```diff
@@ -187,7 +187,6 @@ CHANGELOG
 
 ### Packaging
 - In Debian packages, `intelmqctl check` and `intelmqctl upgrade-config` are executed in the postinst step (#1551, PR#1624 by Birger Schacht).
-- Require `requests<2.26` for Python 3.5, as 2.25.x will be the last release series of the requests library with support for Python 3.5.
```

Comment:
Why are you dropping support for Python 3.5 here? (and if we do so, it needs much more changes)